src/core/InternalAnimatedValue.js

```diff
@@ -13,17 +13,24 @@ function sanitizeValue(value) {
  * were caused by imperative setValue and interpolate – they are currently exposed with AnimatedValue.js
  */
 export default class InternalAnimatedValue extends AnimatedNode {
+  _offset = 0;
   constructor(value) {
     super({ type: 'value', value: sanitizeValue(value) });
     this._startingValue = this._value = value;
     this._animation = null;
   }
 
   __detach() {
-    ReanimatedModule.getValue(
-      this.__nodeID,
-      val => (this.__nodeConfig.value = val)
-    );
+    if (ReanimatedModule.getValue) {
+      ReanimatedModule.getValue(
+        this.__nodeID,
+        val => (this.__nodeConfig.value = val)
+      );
+    } else {
+      setTimeout(() => {
+        this.__nodeConfig.value = this.__getValue();
```

Comment:
not sure if this is needed at all, if so you'd likely need to reset offset to 0 here to. Also not sure if `setTimeout` is required. But in general I think no action is needed